backend/src/test/java/com/votogether/domain/post/controller/PostControllerTest.java

```diff
@@ -871,4 +875,29 @@ void update() throws IOException {
                 .status(HttpStatus.OK);
     }
 
+    @Test
+    @DisplayName("인기 게시물 랭킹을 불러온다.")
+    void getRanking() {
+        // given
+        Post post = Post.builder()
+                .postBody(PostBody.builder().title("제목").build())
+                .writer(MemberFixtures.MALE_10.get())
+                .build();
+        ReflectionTestUtils.setField(post, "id", 1L);
+
+        PostRankingResponse postRankingResponse = new PostRankingResponse(PostCompactResponse.of(post), 1);
+        given(postService.getRanking()).willReturn(List.of(postRankingResponse));
+
+        // when, then
+        List<PostRankingResponse> result = RestAssuredMockMvc.given().log().all()
+                .when().get("/posts/ranking/popular/guest")
+                .then().log().all()
+                .status(HttpStatus.OK)
+                .extract()
+                .as(new ParameterizedTypeReference<List<PostRankingResponse>>() {
+                }.getType());
+
+        assertThat(result.get(0)).isEqualTo(postRankingResponse);
```

Comment:
`P2`
```suggestion
        assertThat(result).usingRecursiveComparision().isEqualTo(List.of(postRankingResponse));
```
`result` 자체로 비교하도록 수정해보는건 어떨까요?

Reply:
오 이런방법이,, 또하나 배워갑니다,, 👍